src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -190,6 +190,8 @@
 
     private static final int FLAG_FIN = 1;
     private static final int FLAG_INIT = 2;
+    private static final int FLAG_INCOMPLETE = 4;
+    private static final int FLAG_INIT_INCOMPLETE = 6;
```

Comment:
Suggest making this more readable
```
private static final int FLAG_INIT_INCOMPLETE = FLAG_INIT | FLAG_INCOMPLETE;
```

Similar feedback for `FLAG_INIT_AND_FIN` below.